topics/dot-net/index.md

```diff
@@ -0,0 +1,14 @@
+---
+aliases: dot-net-framework, microsoft-dot-net
+created_by: Microsoft
+display_name: .NET
+logo: dot-net.png
+released: February 13 2002
+short_description: .NET framework is developed by Microsoft which helps developers to create applications more easily.
+topic: dot-net
+related: software, framework
+url: https://microsoft.com/net
+wikipedia_url: https://en.wikipedia.org/wiki/.NET_Framework
+---
+.NET is collection of shared libray codes which can be used by developers to create applications easily without starting from scrach. It supports C#, C++, F#, Visual Basic, and other programming languages. It is supported by Windows operating system only.
```

Comment:
- Do not use the word "codes" here. [A code is not what you think it means](https://en.wikipedia.org/wiki/Code).
- Since the general audience of GitHub includes what one may consider a "developer", it's probably best to exclude the "can be used by developers" part altogether.
- "to create applications easily without starting from scratch" How? You should explain why this is the case. Tell me what .NET does that makes this the case.
- The .NET Framework does not "support" other programming languages. Rather, one can use the .NET Framework with [CLI-compliant](https://en.wikipedia.org/wiki/List_of_CLI_languages) programming languages. [Source](https://en.wikipedia.org/wiki/.NET_Framework).
- The last sentence "It is supported by Windows operating system only." is **not** true at all. Please change it. [Source](https://www.microsoft.com/net/download/core).